examples/oneapi/cpp/source/covariance/cov_dense_biased_batch.cpp

```diff
@@ -27,7 +27,8 @@ int main(int argc, char const *argv[]) {
     const auto input = dal::read<dal::table>(dal::csv::data_source{ input_file_name });
     auto cov_desc = dal::covariance::descriptor{}
                         .set_result_options(dal::covariance::result_options::cov_matrix)
-                        .set_bias(true);
+                        .set_bias(true)
+                        .set_assume_centered(true);
```

Comment:
Shouldn't it be `false`? I think `assume_centered` can be true only when the input data is centered.